### PR TITLE
Fix broken links on available SlotFills

### PR DESCRIPTION
**Pull request by lukecarbis**

### CODE REVIEW

File: docs/designers-developers/developers/slotfills/README.md [hunk 1]
```diff
@@ -99,14 +99,14 @@ function PostStatus( { isOpened, onTogglePanel } ) {
...
 
-* [PluginBlockSettingsMenuItem](./plugin-block-settings-menu-item.md)
-* [PluginDocumentSettingPanel](./plugin-document-setting-panel.md)
-* [PluginMoreMenuItem](./plugin-more-menu-item.md)
-* [PluginPostPublishPanel](./plugin-post-publish-panel.md)
-* [PluginPostStatusInfo](./plugin-post-status-info.md)
-* [PluginPrePublishPanel](./plugin-pre-publish-panel.md)
-* [PluginSidebar](./plugin-sidebar.md)
-* [PluginSidebarMoreMenuItem](./plugin-sidebar-more-menu-item.md)
+* [PluginBlockSettingsMenuItem](./plugin-block-settings-menu-item)
```
Comment (talldan) [inline docs/designers-developers/developers/slotfills/README.md]: Comparing to other links in the handbook, it seems like absolute paths are needs for the handbook to correctly process them.
```suggestion
* [PluginBlockSettingsMenuItem](/docs/designers-developers/developers/slotfills/plugin-block-settings-menu-item.md)
```

**Review by talldan — CHANGES_REQUESTED** [review]
@lukecarbis Thanks for the PR. The trouble with removing the extension is that the links no longer work within github.

I had a look at some other links in the documentation, and it seems as though they all still included the `.md` extension, the handbook must strip this off when generating the docs. An example is the links in the Block API section that work in both places:
https://github.com/WordPress/gutenberg/blob/master/docs/designers-developers/developers/block-api/README.md

The main difference seems to be that all the other links seem to be absolute, so I think changing these to absolute paths will fix the issue.

File: docs/designers-developers/developers/slotfills/README.md (hunk 1 above)
Comment (gziolo) [reply docs/designers-developers/developers/slotfills/README.md]: Yes, this is correct. @lukecarbis - can you update all those links?

Comment (lukecarbis) [reply docs/designers-developers/developers/slotfills/README.md]: @gziolo @talldan Thanks for the tip! Implemented in 5652555.

**Review by talldan — APPROVED** [review]
Looks great, thanks again for taking the time to fix this and congrats on your first merged PR 🎉

**Pull request merged**